frontend/src/metabase/palette/hooks/useCommandPalette.tsx

```diff
@@ -84,17 +86,17 @@ export const useCommandPalette = ({
       limit: 20,
     },
     {
-      skip: !debouncedSearchText || !isSearchTypeaheadEnabled,
+      skip: !debouncedSearchText || !isSearchTypeaheadEnabled || disabled,
       refetchOnMountOrArgChange: true,
     },
   );
 
   const { data: recentItems, refetch: refetchRecents } = useListRecentsQuery();
```

Comment:
this query doesn't take into account `disabled` option and will be fired every time.